libs/application/templates/party-application/src/lib/PartyApplicationTemplate.ts

```diff
@@ -275,12 +275,14 @@ const PartyApplicationTemplate: ApplicationTemplate<
   stateMachineOptions: {
     actions: {
       assignToSupremeCourt: assign((context) => {
+        const supremeCourtIds = process.env.SUPREME_COURT_ASSIGNEES?.split(
```

Comment:
Maybe rename `supremeCourt` to `electoralCommission` since `supremeCourt` is `Hæðstiréttur`